community/dbms/src/main/java/org/neo4j/commandline/dbms/CsvImporter.java

```diff
@@ -145,4 +112,9 @@ public void doImport() throws IOException
         ImportTool.doImport( outsideWorld.errorStream(), outsideWorld.errorStream(), storeDir, logsDir, reportFile, fs,
                 nodesFiles, relationshipsFiles, false, input, config, badOutput, configuration );
     }
+
+    private boolean isIgnoringSomething()
+    {
+        return ignoreBadRelationships | ignoreDuplicateNodes | ignoreExtraColumns;
```

Comment:
Is there a reason why you want to do bitwise OR here instead of `||` Did you want to explicitly avoid short-circuiting?